defaults/main.yml

```diff
@@ -60,3 +60,49 @@ steamengine_java_base_opts:
 steamengine_java_opts:
   - "-Xmx512M"
   - "-Xms512M"
+
+# DRUPAL
+# Path of drush launcher
+steamengine_drush_launcher_path: "/usr/local/bin/drush"
+# Command run at the end of deployment
+steamengine_drupal_drush_command_deployment:
+  - "deploy -y"
+# List of directory we need write permission
+steamengine_drupal_path_with_write_permission:
+  - "web/sites/default/files"
```

Comment:
If this path is configurable through Drupal config (file_public_path) why bothering with a symlink ?

Reply:
Pour que les fichiers restent accessible via le nginx et que le fixpermission l'utilise